server/server.go

```diff
@@ -1516,6 +1522,11 @@ func (s *Server) Start() {
 		s.startWebsocketServer()
 	}
 
+	// MQTT
+	if opts.MQTT.Port != 0 {
```

Comment:
Be good if we could configure this to be the default easily since its already established.

Reply:
No, again, like we do for routes, etc.. this is how the server knows if it should start MQTT, it is by checking if the port is set.